script/post_split.rb

```diff
@@ -0,0 +1,92 @@
+new_subject = 'test subject'
+reply_user = 'Throne3d'
+reply_id = 1024 # we split *after* this reply_id
```

Comment:
You probably want a way to set these when launching the script? also why are you setting reply_user instead of just using the reply's user?